docker/seccomp.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+def loadDefault( filename ):
+    with open( filename, 'r' ) as f:
+        return json.load( f )
+
+def inPlaceTransform( jdata ):
+    # for names_action_etc in jdata[ 'syscalls' ]:
+    #     names = names_action_etc[ 'names' ]
+    #     action = names_action_etc[ 'action' ]
+    #     if 'personality' in names:
+    #         print( action )
+    jdata[ 'syscalls' ].extend(
+        { 
+            "names": [ "personality" ],
+            "action": "SCMP_ACT_ALLOW",
+            "args": [ { "index": 0, "value": value, "op": "SCMP_CMP_EQ" } ]
+        }
+        for value in [ 0x0040000, 0x0400000, 0x0440000 ]
+    )
+
+if __name__ == "__main__":
+    jdata = loadDefault( sys.argv[1] )
```

Comment:
A boilerplate argparse here would be good. Something like

```
parser = argparser.ArgumentParser(description="Modify default docker seccomp profile to allow syscalls needed by poplog")
parser.add_argument("docker_seccomp_json", type=Path, help="Path to docker's default seccomp profile")

def main(argv=sys.argv):
    args = parser.parse_args(argv)
    jdata = loadDefault(args.docker_seccomp_json)
    inPlaceTransform(jdata)
    json.dump(jdata, sys.stdout, indent=4)
    print()

if __name__ == '__main__':
    main()
```

Reply:
Nice one. I wanted to do argparse but it was too much like hard work. Now I shall steal this code ....

Reply:
And added.